src/p2p/Join/routes.ts

```diff
@@ -223,14 +223,18 @@ const unjoinRoute: P2P.P2PTypes.Route<Handler> = {
       return res.status(500).send(processResult.error)
     }
 
-    // we need to remove the unjoin request this cycle since we are waiting until next cycle to gossip it to the network
-    // processNewUnjoinRequest automatically adds the unjoin request to newUnjoinRequests it will be added back to our list
-    // once we process the queued request next cycle
-    removeUnjoinRequest(unjoinRequest.publicKey)
-
-    queueUnjoinRequest(unjoinRequest)
-
-    return res.status(200).send()
+    Comms.sendGossip(
+      'gossip-unjoin',
+      unjoinRequest,
+      '',
+      null,
+      nodelistFromStates([
+        P2P.P2PTypes.NodeStatus.ACTIVE,
+        P2P.P2PTypes.NodeStatus.READY,
+        P2P.P2PTypes.NodeStatus.SYNCING,
+      ]),
+      true
+    )
```

Comment:
whats up with this change? why was it made? it'll probably break unjoin

Reply:
You're right. I was looking at `serviceQueue-impl` branch to double check this earlier. It has gossip still. I just checked dev (which I should have just referenced this diff). Changed back and resolving this comment after committing.